ra/registration-authority.go

```diff
@@ -787,7 +789,40 @@ func (ra *RegistrationAuthorityImpl) UpdateAuthorization(base core.Authorization
 	}
 
 	// Dispatch to the VA for service
-	ra.VA.UpdateValidations(authz, challengeIndex)
+
+	doValidate := func() {
+		records, err := ra.VA.PerformValidation(authz.Identifier.Value, authz.Challenges[challengeIndex], authz)
+		var prob *probs.ProblemDetails
+		if p, ok := err.(*probs.ProblemDetails); ok {
+			prob = p
+		} else if err != nil {
+			// TODO something more sensible than this, please
```

Comment:
Set `prob` to a internal server problem with details 'uncaught problem' or something.